save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/vulnerability/VulnerabilityController.kt

```diff
@@ -294,4 +294,26 @@ class VulnerabilityController(
     }.map {
         ResponseEntity.ok("Project $projectName was successfully deleted in vulnerability $vulnerabilityName")
     }
+
+    @PostMapping("/save-user")
+    @Operation(
+        method = "POST",
+        summary = "Save new date.",
+        description = "Save new date.",
+    )
+    @PreAuthorize("permitAll()")
+    @ApiResponse(responseCode = "200", description = "Successfully saved new user in vulnerability")
+    @ApiResponse(responseCode = "404", description = "Could not either find user or find vulnerability")
+    @ApiResponse(responseCode = "409", description = "Could not save user as it is already present in vulnerability")
+    fun saveUser(
```

Comment:
`deleteUser` should also be present

Reply:
`getUsers` might be nice too

Reply:
The user is in VulnerabilityDto, so we do not need the get method

Reply:
oh ok

But `deleteUser` is still needed

Reply:
Added DeleteUser